content/SCALE/SCALETutorials/Apps/Collabora.md

```diff
@@ -51,4 +56,7 @@ After a few minutes, the Collabora container displays as **ACTIVE**.
 
 After it does, you can click **Web Portal** to access the admin console.
 
-![Collabora8](/images/SCALE/Collabora8.png "Collabora Admin Console")
\ No newline at end of file
+![Collabora8](/images/SCALE/Collabora8.png "Collabora Admin Console")
+
+{{< taglist tag="scalecollabora" limit="10" >}}
```

Comment:
As of yet, no additional collabora links display under "Related Content," this may not be an issue.